### PR TITLE
Fix song select beatmap panels sometimes not displaying correct background shown in web

### LINKED ISSUE #7524: The mapset thumbnail in song select list is incorrent when the mapset has multiple backgrounds
**Describe the bug:**
When a mapset have multiple backgrounds in different difficulties (for example, No Title https://osu.ppy.sh/beatmapsets/320118#osu/714001 and The Big Black https://osu.ppy.sh/beatmapsets/41823#osu/131891), the mapset thumbnail shown in the song select list is different from the thumbnail on the website or in osu!stable. I think it would be better to keep consistent with the website/stable since some mapsets' backgrounds are iconic.

**Screenshots or videos showing encountered issue:**
For example, the thumbnail of The Big Black is the bg of its taiko diff, but not the well-known one.
osu_2020-01-14_00-30-22

**osu!lazer version:** 2020.112.0

### PR DESCRIPTION
**Pull request by Joehuu**
- Fixes https://github.com/ppy/osu/issues/7524

Tested with:
- https://osu.ppy.sh/beatmapsets/1324491#osu/2743549
- https://osu.ppy.sh/beatmapsets/932291#osu/2166563
- https://osu.ppy.sh/beatmapsets/41823#osu/131891

| Before | After | Web |
| --- | --- | --- |
| VF2t5VGbjU | bLXdAHvmos | firefox_Oy7k69COKx |

### CODE REVIEW

File: osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs [hunk 1]
```diff
@@ -108,7 +108,7 @@ protected override void UpdateItem()
 
             Header.Children = new Drawable[]
             {
-                background = new DelayedLoadWrapper(() => new SetPanelBackground(manager.GetWorkingBeatmap(beatmapSet.Beatmaps.FirstOrDefault()))
+                background = new DelayedLoadWrapper(() => new SetPanelBackground(manager.GetWorkingBeatmap(beatmapSet.Beatmaps.MinBy(b => b.OnlineID)))
```
Comment (Joehuu) [inline osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs:111]: Behavior defined in https://osu.ppy.sh/community/forums/topics/1722422?n=2. Doesn't seem to be documented in wiki https://osu.ppy.sh/wiki/en/Beatmap/Background.

Comment (peppy) [reply osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs:111]: The alphabetical sorting is likely a side effect of something (import process order or realm logic) but isn't something we guarantee.

What you have seems fine as long as it's double-checked against stable code. I'll do this if no one else beats me.

Comment (bdach) [reply osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs:111]: I tried to have a quick dig in stable source but it looks as if the factor determining what image to show in stable is whatever is hosted at `https://b.ppy.sh/thumb/{beatmap_set_id}.jpg`, and I'm not sure I have the source to check how *that* picks an image, so this is probably one you'll need to poke out of there.

Comment (peppy) [reply osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs:111]: Checked BSS code, it looks to be correct. Weakly enforced sort by implicit primary key of query.

**Review by peppy — APPROVED** [review]

**Pull request merged**